addons/block_code/simple_nodes/simple_scoring/simple_scoring.gd

```diff
@@ -63,12 +63,6 @@ func simple_setup():
 	add_child(right_label)
 
 
-func _exit_tree():
-	for label in _score_labels.values():
-		label.queue_free()
-	_score_labels.clear()
-
```

Comment:
This benefits from [add_child()](https://github.com/endlessm/godot-block-coding/pull/216/commits/ec063a7d67587b7827f4a57820ff1b58df554c08#diff-79391123a25292e6eb53815fe07bfa1e878e105297ed83e694c14fe598a63029R55-R63).